test/files/neg/t12851b.check

```diff
@@ -0,0 +1,9 @@
+C_2.scala:2: warning: method (method f in trait T2) with a single empty parameter list overrides method without any parameter list
```

Comment:
This `method (method ...)` repetition always shows up when there's the location string

Reply:
agree, tweak-worthy.